docs/source/samplers.rst

```diff
@@ -86,3 +86,22 @@ High-Volubility sampler
 .. clidoc::
 
    child-project sampler /path/to/dataset /path/to/destination high-volubility --help
+
+Conversation sampler
+~~~~~~~~~~~~~~~~~~~~
+
+The conversation sampler returns the conversational blocks with the highest amount of turns.
+The first step is the detection of conversational blocks.
+Two consecutive vocalizations are considered part of the same conversational block if they are not separated
+by an interval longer than a certain duration, which by default is set to 1000 milliseconds.
+
+Then, the amount of conversational turns (between the key child and adult) is calculated for each conversational block.
+The sampler returns, for each unit, the desired amount of conversations with the higher amount of turns.
```

Comment:
ah, I read the code above as suggesting that I can select only FEM-CHI blocks -- did I get that wrong?

Reply:
you can limit the search to FEM vs CHI only, but by default the search includes all both FEM and MAL. It is more clear now ?

Reply:
Then, the amount of conversational turns (between the key child and adult) --> Then, the amount of conversational turns (by default, between the key child and female/male adult)